.github/workflows/formatting.yaml

```
@@ -41,3 +54,34 @@ jobs:
       if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }} 
       with:
         repo-token: ${{secrets.GITHUB_TOKEN}}
+
+  Build:
```

```suggestion
  BuildAndTest:
```
Since it runs the tests too?

---

Good catch!